mp/src/game/server/neo/neo_player.cpp

```diff
@@ -2505,6 +2505,29 @@ float CNEO_Player::GetSprintSpeed(void) const
 	}
 }
 
+int CNEO_Player::ShouldTransmit(const CCheckTransmitInfo* pInfo)
+{
+	auto player = Instance(pInfo->m_pClientEnt);
+	
+	if(player)
+	{
+		auto neoPlayer = dynamic_cast<CNEO_Player*>(player);
```

Comment:
Could `neoPlayer` be NULL here? If `neoPlayer` is expected in this scope, assign and check `neoPlayer` as the scope's if condition, otherwise check `neoPlayer` when getting ghost.